dags/remote_automated_ampliseq.py

```diff
@@ -0,0 +1,373 @@
+            remote_tar_path = os.path.join(REMOTE_PATH, study['study'], tar_filename)
+            command = f'mkdir -p {REMOTE_PATH}/{study["study"]} && tar --strip-components=1 -xzf {remote_tar_path} -C {REMOTE_PATH}/{study["study"]} && rm {remote_tar_path}'
+            ssh_op = CustomSSHOperator(
+                task_id='extract_remote_directory_' + study['study'],
+                ssh_hook=SSHHook(
+                    remote_host=REMOTE_HOST_SSH,
+                    username=USERNAME,
+                    key_file=KEY_FILE
+                ),
+                command=command
+            )
+            ssh_op.execute(context={})
+
+        @dag.task(trigger_rule=TriggerRule.ALL_SUCCESS)
+        def editing_remote_samplesheet(study):
+            logging.info(f"Editing remote samplesheet for study {study['study']}...")
+            command = f"sed -i 's|{BASE_PATH}/|{REMOTE_PATH}/|g' {REMOTE_PATH}/{study['study']}/samplesheet.csv"
+            ssh_op = CustomSSHOperator(
+                task_id='editing_remote_samplesheet_' + study['study'],
+                ssh_hook=SSHHook(
+                    remote_host=REMOTE_HOST_SSH,
+                    username=USERNAME,
+                    key_file=KEY_FILE
+                ),
+                command=command
+            )
+            ssh_op.execute(context={})
+
+        @dag.task(trigger_rule=TriggerRule.ALL_SUCCESS)
+        def submit_nextflow_script(script_info):
+            logging.info(f"Submitting Nextflow script for study {script_info['study']['study']}...")
+            study = script_info["study"]
+            script_name = script_info["script_name"]
+            command = f'bsub -n 64 -M 250000 -R "rusage[mem=250000] span[hosts=1]" sh {REMOTE_PATH}/{study["study"]}/{script_name}'
+            return CustomSSHOperator(
+                task_id='submit_nextflow_script_' + study['study'],
+                ssh_hook=SSHHook(
+                    remote_host=REMOTE_HOST_SSH,
+                    username=USERNAME,
+                    key_file=KEY_FILE
+                ),
+                command=command,
+            ).execute(context={})
+
+        @dag.task(trigger_rule=TriggerRule.ALL_SUCCESS)
+        def monitor_status_nextflow(study):
+            logging.info(f"Monitoring Nextflow job status for study {study['study']}...")
+            job_sensor = JobStatusSensor(
+                task_id=f"monitor_status_nextflow_{study['study']}",
+                remote_host=REMOTE_HOST_SSH,
+                username=USERNAME,
+                key_file=KEY_FILE,
+                poke_interval=POKE_INTERVAL,
+            )
+            return job_sensor.execute(context={})
+
+        @dag.task(trigger_rule=TriggerRule.ALL_SUCCESS)
+        def compress_remote_results(study):
+            logging.info(f"Compressing remote results for study {study['study']}...")
+            command = f'tar -czf {REMOTE_PATH}/{study["study"]}_results.tar.gz -C {REMOTE_PATH}/{study["study"]} out'
```

Comment:
so its worth considering what all files you actually need from this directory. its likely less a big deal for ampliseq than the shotgun metagenomics stuffs.. but sometimes these directories get quite large and contain a lot of intermediate files that we dont necessarily need. i would consider checking the post processing Rscript for just whats needed and copying that + multiqc